polonius-engine/src/output/mod.rs

```diff
@@ -76,6 +76,7 @@ impl ::std::str::FromStr for Algorithm {
 #[derive(Clone, Debug)]
 pub struct Output<T: FactTypes> {
     pub errors: FxHashMap<T::Point, Vec<T::Loan>>,
```

Comment:
Is this the point where we bite the bullet and rename errors, or do we do that further along?

Reply:
yeah good question

Reply:
I say do it

Reply:
Let's do this at the same time we continue The Great Renaming of 2019 :)